src/display/lv_display.c

```diff
@@ -844,6 +844,15 @@ lv_timer_t * _lv_display_get_refr_timer(lv_display_t * disp)
     return disp->refr_timer;
 }
 
+void _lv_display_delete_refr_timer(lv_display_t * disp)
```

Comment:
```suggestion
void lv_display_delete_refr_timer(lv_display_t * disp)
```

Reply:
I think it's corresponding to `_lv_display_get_refr_timer `.

Reply:
> I think it's corresponding to `_lv_display_get_refr_timer `.

yes.
why ‘_lv_display_get_refr_timer’ have  '_'  ?

Reply:
@kisvegabor 
So, remove `_` from  `_lv_display_delete_refr_timer` and `_lv_display_get_refr_timer` or keep it ?

Reply:
Please remove from both.  `lv_indev_get_read_timer` is without `_` as well.

Reply:
> Please remove from both. `lv_indev_get_read_timer` is without `_` as well.

OK.